src/juju_doctor/probes.py

```diff
@@ -65,6 +55,8 @@ class Probe:
 
     path: Path  # relative path in the temporary folder
     probes_root: Path  # temporary folder
+    probes_chain: str = ""  # probe call chain with format /uuid/uuid/uuid
+    uuid: UUID = field(default_factory=uuid4)
```

Comment:
- We should document fields [per spec](https://stackoverflow.com/a/6061254/3516684).
- Add comment about the use case of uuid (subtree merging).